### ISSUE TITLE
How to upgrade?

### ISSUE THREAD

**Issue by comminutus** [kube-vip/kube-vip-cloud-provider#83]
Hi there, I just noticed a new version of `kube-vip-cloud-provider` was released.  How can I upgrade my existing deployment?

**Comment by Jamison1** [comment]
Also, how does this affect the helm chart?

**Comment by lubronzhan** [comment]
Hi @Jamison1 which helm chart you are referring to?

**Comment by lubronzhan** [comment]
Hi @comminutus please hold, we just discovered a bug in 0.0.8.
Let me fix it and make a new patch today

**Comment by lubronzhan** [comment]
Usually there is no manifest change, you can just update your manifest with new image version

**Comment by comminutus** [comment]
@lubronzhan you mean something like: `kubectl -n kube-system edit deploy kube-vip-provider` and change the `image` value from: `ghcr.io/kube-vip/kube-vip-cloud-provider:v0.0.7` to `ghcr.io/kube-vip/kube-vip-cloud-provider:v0.0.8` (or whatever the latest version is) ?

**Comment by Jamison1** [comment]
@lubronzhan I actually had to build my own helm chart [due to this issue. ](https://github.com/kube-vip/helm-charts/issues/25#issue-1926737436) My comment about how 0.0.8 affects the helm chart references the cloud provider helm chart.  I tried just swapping out the image in the cloud provider helm chart and had issues.  No time lately to debug it.

**Comment by lubronzhan** [comment]
yes @comminutus , or just kubectl apply -f https://github.com/kube-vip/kube-vip-cloud-provider/blob/main/manifest/kube-vip-cloud-controller.yaml once we bumped to 0.0.9

@Jamison1 it should affect helm chart as well. it might not affect existing svc, but it will affect service you deploy in the future. 
Unless you set `KUBEVIP_NAMESPACE` in the config to some hardcoded namespace

**Comment by comminutus** [comment]
@lubronzhan odd, I get this:
```
k apply -f https://github.com/kube-vip/kube-vip-cloud-provider/blob/main/manifest/kube-vip-cloud-controller.yaml
error: error parsing https://github.com/kube-vip/kube-vip-cloud-provider/blob/main/manifest/kube-vip-cloud-controller.yaml: error converting YAML to JSON: yaml: line 203: mapping values are not allowed in this context
```

**Comment by lubronzhan** [comment]
Oh sorry my bad, it should be the raw code

so 
```
kubectl apply -f https://raw.githubusercontent.com/kube-vip/kube-vip-cloud-provider/main/manifest/kube-vip-cloud-controller.yaml
```
